app/css/dialog.scss

```diff
@@ -0,0 +1,74 @@
+// These styles are only use for the privacy modal. This is meant to look like sscom.
+
+$darkBlue: #142237;
+$brightBlue: #005B94;
+
+.sscom.modal {
+  text-align: center;
+
+  .modal-content {
+    border: none;
+  }
+
+  .modal-header {
+    background: $darkBlue;
+    height: 6rem;
+    padding: 0;
+    align-items: center;
+    justify-content: center;
+    border: none;
+
+    h1 {
+      width: 100%;
+    }
+
+    img {
+      margin: auto;
+      width: auto;
+      height: 2rem;
+      padding: 0 1rem;
+    }
+  }
+
+  .modal-body {
+    padding: 2rem 1rem;
+    font-family: Metropolis, 'Proxima Nova', Arial, sans-serif;
```

Comment:
Is there a reason you didn't use `$font-family-sans-serif`? It's got different fallback fonts, but the first two are the same